superset-frontend/src/SqlLab/components/ScheduleQueryButton.tsx

```diff
@@ -179,7 +180,7 @@ const ScheduleQueryButton: FunctionComponent<ScheduleQueryButtonProps> = ({
       </Row>
       {scheduleQueryWarning && (
         <Row>
-          <Col md={12}>
+          <Col xs={24}>
```

Comment:
Not sure we even need these either. Not saying it should affect this PR, just something to consider.